### DISCUSSION TITLE
RollupError: Expected '{', got 'interface'

### DISCUSSION THREAD

**Discussion by gtalk-renato** [rollup/plugins#1710] [Q&A]
I'm developing a project in Vue 3. Everything is working fine in dev-time. But the error RollupError: Expected '{', got 'interface' occurs when I try to build. I've tried everything and I don't know what else to do.
--
Here is a part of the code.
[rollup-problem.zip](https://github.com/rollup/plugins/files/15052296/rollup-problem.zip)

**Comment by shellscape** [comment]
as a general rule of thumb, most open source developers won't download zip files because they can contain dangerous things. you would be better off using a repo, stack blitz or replit to share a reproduction of the problem you're running into.

**Reply by gtalk-renato** [reply]
ok, I'll put a repo
https://github.com/gtalk-renato/rollup-problem

**Comment by LaserKaspar** [comment]
@gtalk-renato Could you post the full error? I am facing the same issue (see: https://github.com/rollup/plugins/issues/1690) and am trying to find similarities in our projects
